README.md

```diff
@@ -57,6 +57,15 @@ collection.Scan(scan => scan
             .As<IScopedService>()
             // And again, just specify the lifetime.
             .WithScopedLifetime());
+        // Generic interfaces are also supported too, e.g. public interface IOpenGeneric<T> 
+        .AddClasses(classes => classes.AssignableTo<IOpenGeneric<>>())
```

Comment:
This syntax doesn't work though. Open generics needs to be registered using a type instance, like this:

```csharp
.AddClasses(classes => classes.AssignableTo(typeof(IOpenGeneric<>)))
```

Reply:
Ah man, sorry about that. I stopped using generics in my project so I hadn't actually tested if it works.

I'm a bit lost now, I created this PR using GitHub's in-browser editor and I don't know how I can make changes to it. When I look at my fork of Scrutor it doesn't have my change in it either.

Think I'll just abandon it!

Reply:
Just add your changes to https://github.com/mattfrear/Scrutor/tree/patch-1 😄 We can squash it when we merge.